code/modules/mining/remote_fob.dm

```diff
@@ -0,0 +1,329 @@
+/////////For remote construction of FOB using a computer on the ship during setup phase
+GLOBAL_LIST_INIT(dropship_lzs, typecacheof(list(/area/shuttle/drop1/lz1, /area/shuttle/drop2/lz2)))
+
+
+/////////////////////////////// Drone Mob
+
+/mob/camera/aiEye/remote/fobdrone
+	name = "Remote Construction Drone"
+	mouse_opacity = MOUSE_OPACITY_OPAQUE
+	icon = 'icons/Marine/remotefob.dmi'
+	icon_state = "drone"
+	var/area/starting_area
+	use_static = FALSE
+	var/turf/spawnloc
```

Comment:
Null reference variables such as `starting_area` and `spawnloc` on `Destroy()` even if you don't intend for the object to be destroyed.
Having loose references causes issues on garbage collecting, and things may change in the future on whether they are able to be deleted or not.

This is a personal preference, but I tend to place the redefined variables first, and then the ones defined for the object, for clarity. Up to you if you want to do it differently.